helm/openwhisk/values.yaml

```diff
@@ -242,14 +254,14 @@ providers:
   db:
     external: false
     # Define the rest of these values if you are using external couchdb instance
-    # host: "10.10.10.10"
-    # port: 5984
-    # protocol: "http"
-    # username: "admin"
-    # password: "secret"
+    host: "10.10.10.10"
+    port: 5984
+    protocol: "http"
+    username: "admin"
+    password: "secret"
   # Alarm provider configurations
   alarm:
-    enabled: false
+    enabled: true
```

Comment:
@csantanapr has mentioned in the past (#352) that is preferable to not enable providers by default here.  Change default in downstream project instead?

Reply:
@csantanapr @dgrove-oss Is there a reason to have the providers switched off by default? I thought it would be much easier for users to get started if things worked out of the box.

Reply:
Given we are switching back to docker, and we do not have an automated way to enable them using docker container factory. I'll disable them for now.. We can review this again once #370 is merged in.